scripts/disk_check.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+What:
+    There have been cases, where disk turns RO due to kernel bug.
```

Comment:
>RO [](http://example.com/codeflow?start=44&length=2)

The comment block is very helpful. To make it better, try to explain RO, or just use readonly.

Reply:
done.